acs-category-theory-notes.tex

```diff
@@ -142,20 +198,20 @@ \subsection{Adding an element to a set}
 
 \subsubsection*{Initial (Universal) Solution}
 
-\emph{Initial}: Any other solution can be mapped from this solution.
+An \emph{initial} solution is one that is a suitable domain to map to any other
+solution. Note that such a solution doesn't always exist. In the case of adding
+a single element to a set, the problem can be specified in terms of finding an
+appropriate $S \overset{f}{\rightarrow} T \ni t$ such that
 
-Note that the solution doesn't always exist.
-
-Find $S \overset{f}{\rightarrow} T \ni t$ such that
-$$
-\forall S \overset{f'}{\rightarrow} S' \ni x \ .\ 
-\exists!\ h : T \rightarrow S' \text{ such that }
-h(t) = x \text{ and }
-h \circ f = f'
-$$
+\begin{equation*}
+  \forall S \overset{f'}{\rightarrow} S' \ni x \ .\ 
+  \exists!\ h : T \rightarrow S' \text{ such that }
+  h(t) = x \text{ and }
+  h \circ f = f'
+\end{equation*}
 
-Solution
-$$
```

Comment:
For some reason I thought tikzcd had to be used inside the math environment... I guess I was mistaken. (and indeed it still renders the diagram using mathmode symbols.)